### ISSUE TITLE
rainbow mode!

### ISSUE THREAD

**Issue by colemickens** [lhvy/pipes-rs#102]
Hi,

First of all, I love this, thank you for making it. We all deserve nice things in Rust and what better than `pipes.sh`!

That having been said, I recently saw a screenshot of a particular wayland compositor that features pipes with rainbow colors.

This is the link to that screenshot: https://camo.githubusercontent.com/7481081d2b3d79ed59007f1978b7364da1c503c0c767f0131344da7728428fb6/68747470733a2f2f692e696d6775722e636f6d2f427058784d38482e706e67

tl;dr: `-c rgb` is sweet, but can we have a `-c rainbow`  too?!

**Comment by lhvy** [comment]
Thanks for the suggestion! No guarantees but I'll take a look at this soon.

**Comment by colemickens** [comment]
I'm already having a cheeky day and this is making me grin even more. <3 Thank y'all!

**Comment by colemickens** [comment]
And an automatic new release? :heart_eyes:
